app/Plugin/OidcAuth/Lib/Oidc.php

```diff
@@ -302,9 +302,10 @@ private function prepareClient()
         $providerUrl = $this->getConfig('provider_url');
         $clientId = $this->getConfig('client_id');
         $clientSecret = $this->getConfig('client_secret');
+        $issuer = $this->getConfig('issuer', $providerUrl);
```

Comment:
It is no need to use second argument `$providerUrl` is if `issuer` is not set, `$issuer` will be `null` as OpenIDConnectClient library expected.

Reply:
You are correct, merged.